agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/mustering-game-state/player-mustering-game-state/PlayerMusteringGameState.ts

```diff
@@ -22,7 +22,7 @@ import Order from "../../../game-data-structure/Order";
 
 type MusteringRule = (Mustering & {cost: number});
 
-export type Mustering = {from: Unit | null; region: Region; to: UnitType};
+export type Mustering = {from: Unit | null; region: Region; to: UnitType, affectedUnit?: Unit};
```

Comment:
And last but not least the `affectedUnit` to the Mustering type for easier removing a mustering by clicking the mustered unit.